### ISSUE TITLE
Can't reproduce fcis_end2end_train_test.py with this repo's denmark data?

### ISSUE THREAD

**Issue by rbavery** [chrisckri/InstanceSegmentation_Sentinel2#14]
I'm running into issues [in the fcis training script](https://github.com/msracver/FCIS/blob/master/experiments/fcis/fcis_end2end_train_test.py), where the shape of the rpn class score is not inferred correctly by the [`infer_shape` function](https://github.com/msracver/FCIS/blob/309f6c9a1669bf05a955b66519a807951e70197f/fcis/train_end2end.py#L103) (this function eventually calls a compiled mxnet function). The error is `infer_shape expects (1, 17328) for the region proposal class score but receives (12996)` The result of this sets up the shape of the weights that are initialized in the network.

I think the error may be related to the version of mxnet (tested with .11 and .12) that I'm using since I've running it from docker images and this repo ran from particular commits. But before I try again to install a 2 year old version of mxnet from source, has anyone else has gotten FCIS to work in training mode (not evaluation mode)? what mxnet version was used? has anybody else seen issues with the shape being inferred incorrectly and have thoughts on why this could be the case? 

I also tried to setup the docker environment with mxnet 1.4.1 which was specified in this repo's requirements.txt, but I get an error that the nvcc binary cannot be found when compiling the FCIS operators, which makes me think that this was not the version used to generate the results.

The reason I'm trying to reproduce this repo is that the paper's results show a substantial improvement over my own tests with detectron2's MaskRCNN, which has a lot of false negatives. I've gotten detectron2 running on this denmark dataset with as close of hyperparameters as I can as to what was used for FCIS, and I still observe performance worse than what is reported in the paper.

**Comment by rbavery** [comment]
Update, by changing the NUM_ANCHORS: 12 in the config from 12 to 9 I'm able to train the model from these model weights referenced in a previous issue https://drive.google.com/file/d/1Y62BerMWV_C67oapOIXUzY0yK8XR7zRN/view?usp=sharing

I'm not sure why I need to change the number of anchors though, it might be that the config file in this repo doesn't match the available weights.
